tests/dummy/config/environment.js

```diff
@@ -20,6 +20,11 @@ module.exports = function (environment) {
     },
     'ember-cli-notifications': {
       autoClear: false
+    },
+    'ember-prop-types': {
+      spreadProperty: 'options',
+      throwErrors: true,
+      validateOnUpdate: false
```

Comment:
Configuring `ember-prop-types` to throw errors instead of log warnings which causes tests to crash and burn if they aren't following proper component API's. Also configures `ember-prop-types` to work with `ember-spread` by evaluating properties passed in under the `options` spread property.